run.go

```diff
@@ -108,6 +108,9 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	if spec.Process.Cwd == "" {
 		spec.Process.Cwd = DefaultWorkingDir
 	}
+	if err := os.MkdirAll(filepath.Join(mountPoint, b.Workdir), 0755); err != nil {
+		return fmt.Errorf("error ensuring working directory %q exists: %v)", b.Workdir, err)
```

Comment:
I know it's unlikely we'll ever hit this, but do we need differing verbiage between these two functions so we could tell more easily where the error is if it ever does pop?    Perhaps just changing ' exists:' to ' exists in Run():'.  Not a biggy either way, thought I'd throw it out there for consideration.